### PR TITLE
Update the content

### PR DESCRIPTION
**Pull request by shavidissa**
- Update content
- Add the new image
- Remove the metrics.md file and point to the metrics.md file in the sdk_source_docs repository.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,36 +1,156 @@
...
+
+This is the Wavefront by VMware OpenTracing SDK for C# that provides distributed tracing support for Wavefront.
+
+**Before you start implementing, let us make sure you are using the correct SDK!**
+
+![Java Tracing SDK Decision Tree](docs/csharp_tracing_sdk.png)
+
+>  <b><i>Note</i></b>:
+> </br>
+>  * <b>This is the Wavefront by VMware OpenTracing SDK for c#!</b>
```
Comment (rkempf42) [inline README.md]: C# (not c#)

File: README.md [hunk 2]
```diff
@@ -1,36 +1,156 @@
...
-### Package Manager Console
+## Usage
 
-```
-PM> Install-Package Wavefront.OpenTracing.SDK.CSharp
-```
+[Tracer](https://github.com/opentracing/specification/blob/master/specification.md#tracer) is an OpenTracing [interface](https://github.com/opentracing/opentracing-csharp#initialization) for creating spans and propagating them across arbitrary transports.
 
-### .NET CLI Console
+This SDK provides a `WavefrontTracer` to:
```
Comment (rkempf42) [inline README.md]: This SDK provides a `WavefrontTracer` that:
(or some other way to make the sentence flow. could also change the bullet text).

File: README.md [hunk 3]
```diff
@@ -105,19 +226,23 @@ IReporter compositeReporter = new CompositeReporter(wfSpanReporter, consoleRepor
...
 ITracer tracer = wfTracerBuilder.Build();
 ```
 
 #### Sampling (Optional)
-You can optionally apply one or multiple sampling strategies to the `WavefrontTracer`. See the [sampling documentation](https://github.com/wavefrontHQ/wavefront-opentracing-sdk-csharp/blob/master/docs/sampling.md) for details.
+Optionally, you can apply one or multiple sampling strategies to the `WavefrontTracer`. See the [sampling documentation](https://github.com/wavefrontHQ/wavefront-opentracing-sdk-csharp/blob/master/docs/sampling.md#sampling) for details.
 
 #### Multi-valued Span Tags (Optional)
-You can optionally add metadata to OpenTracing spans in the form of multi-valued tags. The `WavefrontTracer` builder supports different methods to add those tags.
+Optionally, you can add metadata to OpenTracing spans in the form of multi-valued tags. The `WavefrontTracer` builder supports different methods to add those tags.
```
Comment (rkempf42) [inline README.md]: builder or Builder (using Builder above)

File: README.md (hunk 1 above)
Comment (shavidissa) [reply README.md]: Good catch :)

**Pull request merged**